lib/fawry/requests/charge_request.rb

```diff
@@ -5,7 +5,7 @@
 module Fawry
   module Requests
     module ChargeRequest
-      DEFAULTS = { payment_method: 'PAYATFAWRY', currency_code: 'EGP' }.freeze
+      DEFAULTS = { payment_method: 'CARD', currency_code: 'EGP' }.freeze
```

Comment:
I don't think this should be `CARD` by default, but you can provide it as param for card payments